sbt-plugin/src/sbt-test/gen-java/02-server-reflection/src/main/java/example/myapp/helloworld/Main.java

```diff
@@ -36,19 +38,31 @@ public static void main(String[] args) throws Exception {
     public static CompletionStage<ServerBinding> run(ActorSystem sys) throws Exception {
         Materializer mat = SystemMaterializer.get(sys).materializer();
 
+        //#server-reflection-manual-concat
+        // Create service handlers
+        Function<HttpRequest, CompletionStage<HttpResponse>> greetingPartial =
+                GreeterServiceHandlerFactory.create(new GreeterServiceImpl(), "greeting-prefix", sys);
+        Function<HttpRequest, CompletionStage<HttpResponse>> echoPartial =
+                EchoServiceHandlerFactory.create(new EchoServiceImpl(), sys);
+        // Create the reflection handler for multiple services
+        Function<HttpRequest, CompletionStage<HttpResponse>> reflectionPartial =
+                ServerReflection.create(Arrays.asList(GreeterService.description, EchoService.description), sys);
+
+        // Concatenate the partial functions into a single handler
+        ServiceHandler.concatOrNotFound(
+                                greetingPartial,
+                                echoPartial,
+                                reflectionPartial);
+        //#server-reflection-manual-concat
+
         //#server-reflection
         // Instantiate implementation
         GreeterService impl = new GreeterServiceImpl();
 
         // Bind service handler servers to localhost:8080
         return Http.get(sys)
-	  .newServerAt("127.0.0.1", 8080)
-	  .bind(
-            ServiceHandler.concatOrNotFound(
-                GreeterServiceHandlerFactory.create(impl, sys),
-                ServerReflection.create(Arrays.asList(GreeterService.description), sys)
-            )
-	  );
+          .newServerAt("127.0.0.1", 8080)
+          .bind(GreeterServiceHandlerFactory.createWithServerReflection(impl, sys));
```

Comment:
In order to not lose the example on creating the `ServerReflection` given a list of descriptors, I added a new example (lines 41:57) with a more advanced combination of services: (1) a service build with an overload `create` method with (2) a new EchoService and (3) the ServerReflection for those two combined.